### PR TITLE
Adds firefox compatability + more (once manifest v3 is supported)

### PR DESCRIPTION
**Pull request by MidSpike**
# Features

- Firefox support (once Firefox officially supports manifest v3)
- Allows for horizontal scroll wheels (yes I know...) to change volume properly
- Performed a quick lint by hand

# Testing

I've manually tested the unpacked folder on Chrome and zipped it up for Firefox.
Both browsers appear to be working properly.

### CODE REVIEW

**Comment by Araxeus** [comment]
Thanks for the firefox improvement, but I just don't understand what's the point of the horizontal scroll?
from what I know it can only be done with SHIFT + scroll wheel, which is already a function in this extension (x2 volume steps)

**Comment by MidSpike** [comment]
My mouse, and a few others allow you to rock the scroll wheel, left and right.
So it acts as a native horizontal scroll wheel.

File: unpacked/pageAccess.js [hunk 1]
```diff
@@ -40,14 +40,16 @@ setInterval(() => window._lact = Date.now(), 9e5);
     $('.html5-video-player')
 ).onwheel = event => {
     event.preventDefault();
-    // Event.deltaY < 0 means wheel-up
```
Comment (Araxeus) [inline unpacked/pageAccess.js:43]: Please don't delete comments

**Comment by Araxeus** [comment]
LGTM thanks 👍🏼

**Comment by Araxeus** [comment]
I will soon release a new version on the chrome-store, and also consider publishing it on addons.mozilla.org

**Comment by Araxeus** [comment]
https://addons.mozilla.org/en-US/firefox/addon/youtube-volume-scroll/

Installing manifest v3 add-ons will be available on stable on 17.01.2023 (tomorrow at the time of writing)

**Pull request merged**